lib/src/Base/Func/DualLinearCombinationEvaluation.cxx

```diff
@@ -118,37 +121,58 @@ String DualLinearCombinationEvaluation::__str__(const String & ) const
         if (first) oss << value;
         else if (value > 0.0) oss << " + " << value;
         else oss << " - " << -value;
-        first = false;
```

Comment:
This removal is because the case where the expression is first is already taken into account in the `if (mustPrint)` segment at line 133 below. This is why this code can be removed without any change in the produced output.

Reply:
That comment is for the record and this is why I resolve it.